ext/cuda/remapping_distributed.jl

```diff
@@ -275,9 +273,9 @@ function set_interpolated_values_kernel!(
 
         h = local_horiz_indices[i]
         out[i, k] = 0
-        for t in 1:Nq, s in 1:Nq
+        for t in 1:Nq
```

Comment:
Again, is this a bug?

Reply:
Yes, this is also an untested bug.

Reply:
Also, cc @Sbozzolo

Reply:
This should been tested here: https://github.com/CliMA/ClimaCore.jl/blob/a1c6470b8ea8f32143be190feea56e91749e9128/test/Remapping/distributed_remapping.jl#L666-L760

Reply:
I think you meant here:
https://github.com/CliMA/ClimaCore.jl/blob/a1c6470b8ea8f32143be190feea56e91749e9128/test/Remapping/distributed_remapping.jl#L866-L897

This test has been reenabled on GPUs and appears to be passing now.